keras_cv/models/backbones/mix_transformer/mix_transformer_backbone.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MiT backbone model.
+
+References:
+
+"""  # noqa: E501
+
+import numpy as np
+
+from keras_cv import layers as cv_layers
+from keras_cv.backend import keras
+from keras_cv.models import utils
+from keras_cv.models.backbones.backbone import Backbone
+from keras_cv.models.backbones.mix_transformer.mix_transformer_backbone_presets import (  # noqa: E501
+    backbone_presets,
+)
+from keras_cv.models.backbones.mix_transformer.mix_transformer_backbone_presets import (  # noqa: E501
+    backbone_presets_with_weights,
+)
+from keras_cv.utils.python_utils import classproperty
+
+
+@keras.saving.register_keras_serializable(package="keras_cv.models")
+class MiTBackbone(Backbone):
+    def __init__(
+        self,
+        input_shape=None,
+        input_tensor=None,
+        classes=None,
+        include_top=None,
+        embed_dims=None,
+        depths=None,
+        pooling=None,
+        **kwargs,
+    ):
+        if include_top and not classes:
```

Comment:
We dropped `include_top`, `classes`, and `pooling` from the backbone API as the classification task is now handled by the `ImageClassifier` model, which takes a backbone as a parameter.